webserver/src/main/java/io/kestra/webserver/utils/TimeLineSearchUtils.java

```diff
@@ -0,0 +1,64 @@
+package io.kestra.webserver.utils;
+
+import io.kestra.core.models.QueryFilter;
+import lombok.Builder;
+
+import java.time.Duration;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeParseException;
+import java.util.List;
+@Builder
+public record TimeLineSearchUtils(
```

Comment:
```suggestion
public record TimeLineSearch(
```

A record is a plain data carrier, so it's not an Utils which is usually a static class with helper methods.

Reply:
done